### ISSUE TITLE
Detection By Coordinates Issue

### ISSUE THREAD

**Issue by ryansmithdogs** [volkamerlab/teachopencadd#213]
https://github.com/volkamerlab/teachopencadd/blob/03a43302cbf485005a17c5b9a7de13ed529f3ccd/teachopencadd/talktorials/T018_automated_cadd_pipeline/utils/binding_site_detection.py#L51

I ran into an issue when attempting to define the binding site through the use of coordinates in the binding_site_detection.py util under T018. After further investigation, an error stated it was failing on line 51 due to having only 3 positional arguments. We were able to fix this by adding in the "binding_site_output_path=None" to line 53 to accommodate the 4th variable needed to run the string.

```
    def compute_by_coordinates(self, protein_obj, binding_site_specs_obj, binding_site_output_path=None):
```

**Comment by AndreaVolkamer** [comment]
@ryansmithdogs thanks for the pointer.
@corey-taylor could you please have a look into this and add the suggestion to the code?

**Comment by corey-taylor** [comment]
Addressed in https://github.com/volkamerlab/teachopencadd/pull/216

**Comment by dominiquesydow** [comment]
@corey-taylor could you please double-check if this is completely resolved with #216? If yes, please go ahead and close the issue. Thank you!
